weldr/compose.go

```diff
@@ -111,14 +111,14 @@ func (c Client) GetComposeTypes(distro string) ([]string, *APIResponse, error) {
 
 // StartCompose will start a compose of a blueprint
 // Returns the UUID of the build that was started
-func (c Client) StartCompose(blueprint, composeType string, size uint) (string, *APIResponse, error) {
+func (c Client) StartCompose(blueprint, composeType string, size uint) (string, []string, *APIResponse, error) {
```

Comment:
Would it be simpler to add a warnings field to the `APIResponse` instead of returning an extra variable that's rarely used?

Reply:
@bcl ?

Reply:
Happy to change it if it's clearer that way

Reply:
I think extending APIResponse is better. If we change the function signatures we need to bump the API version of the weldr library which is something I want to avoid as long as possible :) Adding fields to structures is safe, thanks to go defaulting to empty values.

It may even be possible to reuse the current `APIErrorMsg` error structure, but I'd have to look closer at all the places that check it and the status value to make sure it doesn't assume an error condition in the wrong situations.

I'd also be tempted to just call it a message, instead of a warning, so that it could be used for other information in the future if needed.